docs/Configuration.md

```diff
@@ -985,31 +985,30 @@ connect to a network resource is the `url` field.
     ]
 }
 ```
+
 #### GBFS Configuration
 
-Steps to add a GBFS feed to a router:
+[GBFS](https://github.com/NABSA/gbfs) is used for a variety of shared mobility services, with partial support for both v1 and v2.2 ([list of known GBFS feeds](https://github.com/NABSA/gbfs/blob/master/systems.csv)).
 
-- Add one entry in the `updater` field of `router-config.json` in the format
+To add a GBFS feed to the router add one entry in the `updater` field of `router-config.json` in the format:
 
 ```JSON
 // router-config.json
 {
-     "type": "bike-rental",
-     "frequencySec": 60,
-     "sourceType": "gbfs",
-     "url": "http://coast.socialbicycles.com/opendata/"
+   "type": "bike-rental",
+   "sourceType": "gbfs",
+   // frequency in seconds in which the GBFS service will be polled
```

Comment:
In case people are copying and pasting this with comments: does the OTP config parser ignore lines starting with `//` or does it stop parsing the file?

Reply:
It works because Jackson is configured to be lenient:
https://github.com/opentripplanner/OpenTripPlanner/blob/b0d17fc94894021e0646161841a5ce9efb7a4dfa/src/main/java/org/opentripplanner/standalone/config/ConfigLoader.java#L49-L50